src/tool_help.c

```diff
@@ -65,6 +65,7 @@ static const char *const helptext[] = {
   "     --crlf          Convert LF to CRLF in upload",
   "     --crlfile FILE  Get a CRL list in PEM format from the given file",
   " -d, --data DATA     HTTP POST data (H)",
+  "     --data-raw      HTTP POST data, '@' allowed (H)",
```

Comment:
Thanks for the tips, Jay. I'll send in another pull request soon that contains the above content.